superflore/generators/ebuild/run.py

```diff
@@ -116,34 +116,48 @@ def main():
                     overlay.repo.get_last_hash()
                 )
             )
+            missing_depends = list()
+            to_commit = list()
+            will_file_pr = False
             for pkg in args.only:
                 if pkg in skip_keys:
                     warn("Package '%s' is in skip-keys list, skipping..."
                          % pkg)
                     continue
                 info("Regenerating package '%s'..." % pkg)
                 try:
-                    regenerate_pkg(
+                    ebuild, deps = regenerate_pkg(
                         overlay,
                         pkg,
                         get_distro(args.ros_distro),
                         preserve_existing
                     )
+                    missing_depends.append(missing_depends)
```

Comment:
This is appending itself to itself and it's never updated. Should it be `deps` that's returned above?

Reply:
@tfoote good point